webapp/src/pages/ResponseList/components/ResponseListForm.js

```diff
@@ -0,0 +1,251 @@
+                val.answers.map(answer => {
+                    // format anwers display based on type
+                    if (selected.includes(answer.question_id)) {
+                        if (answer.type.includes("file")) {
+                            handleAnwsers.push(<a key={answer.value[1]} href={answer.value[0]}>{answer.value[1]}</a>)
+                        }
+                        if (answer.type.includes("choice")) {
+                            let choices = [];
+                            answer.options.map((opt => {
+                                opt.value ? choices.push(<div key={opt.label}><label>{opt.label}</label></div>) : console.log(`${opt.question_id} contains no value`)
+                            }))
+                            handleAnwsers.push(<div key={choices}>{choices}</div>)
+                        }
+                        if (answer.type.includes("text")) {
+                            handleAnwsers.push(<div>
+                                <div key={answer.value} data-tip={answer.value}><p>{answer.value}</p><ReactTooltip
+                                    className="Tooltip"
+                                />
+                                </div>
+                                
+                            </div>)
+                        }
+                    }
+                })
+
+                // extract only the reviewers name
+                val.reviewers.map(review => {
+                    review ? handleReviews.push(review.reviewer_name) : handleReviews.push("")
+                })
+                // envoke and store new columns for Reviews and Answers
+                let addAnwsers = addAnswersCol(val, handleAnwsers);
+                let addReviews = addReviewCol(val, handleReviews);
+
+                // insert anwsers values as columns
+                if (handleAnwsers.length) {
+                    addAnwsers.map((answer, index) => {
+                        let num = (index) + (1);
+                        let key = "Answer" + num;
+                        val[key] = answer
+                    })
+                }
+                // insert new reviews values as columns
+                addReviews.map((review, index) => {
+                    let num = (index) + (1);
+                    let key = "Review" + num;
+                    val[key] = review
+                })
+                // delete original review and answer rows as they don't need to be displayed with all their data
+                handleAnwsers = [];
+                handleReviews = [];
+                delete val.answers;
+                delete val.reviewers;
+            })
+
+            this.setState({
+                responseTable: response,
+                addBtn: { background: "hsl(211deg 100% 50%)" }
+            }
+            )
+
+        })
+    }
+
+
+    generateCol() {
```

Comment:
This looks like it should be `generalCols` because it's generating a list of columns and not a single one?